index.html

```diff
@@ -0,0 +1,305 @@
+             project you need coded, don’t hestiate to contact me.</p>
+
+            <a href="#" class="btn-connect"><h6>Let's connect</h6></a>
+          <ul class="medium">
+              <li class="social-media"><a href="#"><i class="fa fa-twitter"></i></a></li>
+              <li class="social-media"><a href="#"><i class="fa fa-linkedin"></i></a></li>
+              <li class="social-media"><a href="#"><i class="fa fa-medium"></i></a></li>
+              <li class="social-media"><a href="#"><i class="fa fa-github"></i></a></li>
+              <li class="social-media"><a href="#"><i class="fa fa-angellist"></i></a></li>
+          </ul>
+          <br>
+          <a href="#" class="btn-one">Get My Resume</a>
+          </div>
+       </div>
+
+    <div class="text">
+    <ul class="stacks">
+        <li class="languages">
+            
+            <a href="#"> <h2>Languages</h2></a>
+            <ul class="grid-container">
+            <li>
+                
+                <img src="image/js.png" alt="js_logo">
+                <p>JavaScript</p>
+                
+            </li>
+            <li>
+                
+                <img src="image/html.png" alt="html_logo">
+                <p>HTML</p>
+                
+            </li>
+                
+            <li class="langi">
+                
+                <img src="image/css.png" alt="css_logo">
+                <p>CSS</p>
+                
+            </li>
+                
+            
+          </ul>
+        </li>
+        
+        <li>
+            <a href="#"> <h2>Frameworks</h2> </a>
+        </li>
+        
+        <li>
+            
+          <a href="#"> <h2>Skills</h2> </a>
+  
+        </li>
+    </ul>
+    </div>
+</div>
+</div>
+</section>
+
+<section class="contact-wrapper">
+  <section id="contact" class="contacts-form">
+    <h1>Contact me</h1>
+    <p>
+      If you have an application you are interested in developing, a
+      feature that you need built or a project that needs coding. I’d love
+      to help with it
+    </p>
+    <form action="https://formspree.io/f/xgerldko" method="POST">
+      <input
+        type="text"
+        maxlength="30"
+        class="name"
+        name="name"
+        placeholder="Name"
+        required
+      />
+      <input
+        type="email"
+        class="email"
+        name="email"
+        placeholder="Email"
+        required
+        pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$"
+      />
+      <textarea
+        maxlength="500"
+        name="message"
+        id="message"
+        cols="30"
+        rows="7"
+        class="msg"
+        required
+        placeholder="Enter your message here"
+      ></textarea>
+      <button type="submit" class="project-button send">
+        Get in Touch
+      </button>
+    </form>
```

Comment:
- [ ] On desktop size, the form inputs stop to use all the weight screens, look at the original template and please, fix it. 🐱

| original | your project |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/20563252/126810422-a6dbd278-633d-4eff-8119-7d25d2577c99.png) | ![image](https://user-images.githubusercontent.com/20563252/126810463-0808760a-3939-40ec-8f26-092188708e19.png) |